demos/link-prediction/README.md

```diff
@@ -1,62 +1,33 @@
-## Table of Contents
+# Link prediction using StellarGraph
 
-This folder contains several examples of applying graph machine learning (ML) algorithms on network-structured
-data to solve link prediction (predicting relations and/or their properties) problems. The
-examples demonstrate using the `StellarGraph` library to build machine learning
-workflows on both homogeneous and heterogeneous networks.
+[StellarGraph](https://github.com/stellargraph/stellargraph) provides numerous algorithms for doing link prediction on graphs. This folder contains demos of all of them to explain how they work and how to use them as part of a TensorFlow Keras data science workflow.
 
-Each folder contains one or more examples of using the StellarGraph implementations of the
-state-of-the-art algorithms, attri2vec [4], GraphSAGE [3], HinSAGE, GCN [6], GAT [7], Node2Vec [1], and Metapath2Vec [2].
-GraphSAGE, HinSAGE, and GAT are variants of Graph Convolutional Neural networks [6]. Node2Vec and
-Metapath2Vec are methods based on graph random walks and representation learning using the
-Word2Vec [5] algorithm. attri2vec[4] is also based on graph random walks and learns node
-representations by performing a mapping on node attributes.
+A link prediction task predicts an attribute of links/edges in a graph. For instance, predicting whether a link/edge that isn't already in the graph should exist (binary classification, or recommendation, or knowledge base completion, in a knowledge graph), or even labelling existing links with a categorical class (binary classification or multiclass classification), or predicting a continuous number (regression). It is supervised or semi-supervised, where the model is trained using a subset of links/edges that have ground-truth labels. For predicting edge existence, the ground-truth may just be whether the edge exists in the original data, rather than a separate label.
 
-The examples folder structure is shown below.
+Link prediction can also be done as a downstream task from node representation learning/embeddings, by combining node embedding vectors for the source and target nodes of the edge and training a supervised or semi-supervised classifier against the result. Unsupervised algorithms that can be used in this manner include random walk-based methods like Metapath2Vec. StellarGraph provides [demos of unsupervised algorithms](../embeddings).
```

Comment:
I'm not sure on this one, but I think some of these are unsupervised because the edges aren't explicitly labelled - they just already exist in the data.

Reply:
I think that's covered by the last sentence of the previous paragraph?

> For predicting edge existence, the ground-truth may just be whether the edge exists in the original data, rather than a separate label.